tests/integration/pilot/ingress/ingress_test.go

```diff
@@ -283,3 +283,104 @@ spec:
 			}
 		})
 }
+
+// TestIngress tests named-port is converted to the service port value
+func TestIngressWithNamedPort(t *testing.T) {
```

Comment:
Can you just add another case to TestIngress? Otherwise this will spin up a whole new client, etc which has additional cost

Reply:
DONE